main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/224083193-ec1ed8f9-0a99-437e-8eb4-faa89e5fc34a.png)
fix this